src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,300&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Product cards</h1>
+
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__items"
+        src="./images/imac.jpeg"
+        alt="Imac photo"
+        width="160px"
```

Comment:
Inline styles such as `width` and `height` should be avoided in HTML. Instead, these styles should be set in the CSS class for the image.